docs/sources/panels-visualizations/visualizations/node-graph/index.md

```diff
@@ -83,6 +83,12 @@ If a node lacks edge connections, it’s displayed on its own outside of the net
 
 {{< docs/shared lookup="visualizations/panel-options.md" source="grafana" version="<GRAFANA_VERSION>" >}}
 
+## Node graph options
+
+Use the following options to refine your node graph visualization.
+
+- **Zoom mode** - Choose how the node graph should handle zoom/scroll events.
+
```

Comment:
Hello @imatwawana and thanks for the feedback.

I had a look at the other visualizations and discovered that they have they have the section with the Viszualization options first in the UI so I moved the Node graph options to be first as well to align with the standard.

About changing to lower case "g" in "Node graph" it's not possible unfortunately as this string is from the name of the Visualization and it's named "Node Graph". I see we have a mixed standard. Example of the mix "Bar chart" and "XY Chart" 😬

Reply:
In the image in the UI you have in the description, the expandable sections of UI options are as follows:

- Panel options
- Nodes
- Edges
- Node Graph
- Data links 
- etc

So I mean just move this section so that it aligns with the order the order in which these options are presented in the image of the UI in the description image. Does that make sense?

> About changing to lower case "g" in "Node graph" it's not possible unfortunately as this string is from the name of the Visualization and it's named "Node Graph". I see we have a mixed standard. Example of the mix "Bar chart" and "XY Chart" 😬

No worries, thanks for checking!